pkg/daemon/daemon.go

```diff
@@ -281,9 +281,24 @@ func (dn *Daemon) nodeStateChangeHandler(old, new interface{}) {
 	}
 	reqReboot := false
 	reqDrain := false
+	if len(dn.LoadedPlugins) == 0 {
+		err := dn.loadVendorPlugins(newState)
+		if err != nil {
+			glog.Errorf("nodeStateChangeHandler(): failed to load vendor plugin: %v", err)
+			dn.exitCh <- err
+			return
+		}
+	}
+	// Get the latest NodeState
+	latestState, err := dn.client.SriovnetworkV1().SriovNetworkNodeStates(namespace).Get(dn.name, metav1.GetOptions{})
```

Comment:
Can we guarantee `latestState` is the latest? I think there is a chance that daemon writer not updating node state immediately after policy is deleted.

Reply:
All the CR change event will be put in the workqueue, and be processed one after another. So I add a logic to check if the spec of the newState is identical with the latestState, if not, the event will be skipped. It makes sure that only the event latestState will be processed.